### PR TITLE
Rebranding to Microsoft Entra

### PR DESCRIPTION
**Pull request by cilwerner**
## Purpose
Rebranding and changing Azure AD to Microsoft Entra

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: js-spa/App/authConfig.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 /**
  * Configuration object to be passed to MSAL instance on creation. 
  * For a full list of MSAL.js configuration parameters, visit:
- * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md 
+ * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/learn/configuration.md
```
Comment (rwike77) [inline js-spa/App/authConfig.js:4]: Should be "docs" in the URL, not "learn"

File: js-spa/App/authConfig.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
- * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md 
+ * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/learn/configuration.md 
  */
 
 const msalConfig = {
     auth: {
         clientId: 'Enter_the_Application_Id_Here', // This is the ONLY mandatory field that you need to supply.
         authority: 'https://login.microsoftonline.com/Enter_the_Tenant_Info_Here', // Defaults to "https://login.microsoftonline.com/common"
-        redirectUri: '/', // You must register this URI on Azure Portal/App Registration. Defaults to window.location.href e.g. http://localhost:3000/
+        redirectUri: 'Enter_the_Redirect_URI_Here/', // You must register this URI in App Registration on the Microsoft Entra admin center. Defaults to window.location.href e.g. http://localhost:3000/
```
Comment (rwike77) [inline js-spa/App/authConfig.js:11]: "in the app registration"

File: protect-function-app/README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ page_type: sample
 languages:
 - nodejs
 name: Node.js Azure Function protecting HTTP trigger function with Easy Auth and access token scope validation.
-description: This Node.js Azure Function protects its own HTTP Trigger function with Easy Auth and access token scope validation. The code in this sample is used by one or more articles on docs.microsoft.com.
+description: This Node.js Azure Function protects its own HTTP Trigger function with Easy Auth and access token scope validation. The code in this sample is used by one or more articles on learn.microsoft.com.
```
Comment (rwike77) [inline protect-function-app/README.md:6]: Maybe we could add a link here?  https://learn.microsoft.com/entra/identity-platform/ maybe?

**Review by OwenRichards1 — APPROVED** [review]
LGTM - thanks!

**Pull request merged**